config/secrets.yml

```diff
@@ -15,8 +15,19 @@ development:
 
 test:
   secret_key_base: a4601e0fb037b7d1ed534a5d5dd114f47b753c9945757ebe0f9fedff98d41b12741f7dd631e0d9a69cd518623384d871b0387ec5645ccccbbb5d9336ff1cf800
+  sfax_username: sfax_username
+  sfax_api_key: sfax_api_key
+  sfax_encryption_key: sfax_encryption_key
+  sfax_init_vector: sfax_init_vector
+
+from_environment: &deployable_settings
+  sfax_username: <%= ENV['SFAX_USERNAME'] %>
```

Comment:
Nice! Let's do that then and standardize on that.

Maybe add something quick in `CONTRIBUTING.md`, like a "Secret Keys" section so others know where to put them?